lib/rules/color-no-hex/README.md

```diff
@@ -8,9 +8,23 @@ a { color: #333 }
  * These hex colors */
 ```
 
+The `--fix` option on the command line can automatically fix all of the problems reported by this rule if applicable. See the options below.
+
 ## Options
 
-### `true`
+`boolean`: `true|false`
+
+If `false`, the rule is disabled. If `true`, the rule is enabled, but no fix will be performed.
+
+`string`: `rgb|rgba|hsl|hsla`
+
+### `rgb` | `rgba`
+
+`rgb` and `rgba` are the same. If the color format is an hex with transparency (Like #FFFFFFDC), the output will be in rgba format. Otherwise, it will be rgb. There's no way to force one or another.
+
+### `hsl` | `hsla`
+
+`hsl` and `hsla` are the same. If the color format is an hex with transparency (Like #FFFFFFDC), the output will be in hsla format. Otherwise, it will be hsl. There's no way to force one or another.
```

Comment:
Looks like you implement not only `fix`, also you add new options, can we separate PR on two parts?

Reply:
Sure we can, but do you think it makes sense? Because `true` with `fix` will not fix anything, actually. The option is needed to select a format to convert the hex colors to.

Any suggestion on how can we do this?

Reply:
@evilebottnawi No problem. I could make `true` convert hex colors to rgb but that would change the behavior for anyone who already uses this rule, so I opted for this approach: `true` with `fix` will remain doing nothing, as it is right now. Then I introduced `rgba?` and `hsla?` so the user can select their preferred format :)

Reply:
@jeddy3 Hey Richard, thank you for your consideration.

Sure, I'll publish it as a `color-format` plugin and link it here :)

Reply:
@filipekiss Great job with the plugin!

Now that https://github.com/stylelint/stylelint/pull/3322 is merged, I'll close with PR.